expand.cpp

```diff
@@ -144,7 +146,7 @@ namespace Sass {
 
   Statement* Expand::operator()(Media_Block* m)
   {
-    To_String to_string;
+    To_String to_string; // (&ctx); -> src-map test fails!??
```

Comment:
It produces an error with my source-map unit tests, therefore I just marked it with a comment. Will need to check source maps after output rendering is stable.